src/mon/OSDMonitor.cc

```diff
@@ -5596,6 +5596,7 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     if (!newcrush.name_exists(name)) {
       err = -ENOENT;
       ss << "item " << name << " does not exist";
+      goto reply;
```

Comment:
the commit message is misleading. `wait_for_finished_proposal()` does not trigger a new proposal.